test/unit/specs/store/governance/proposals.spec.js

```diff
@@ -171,15 +162,10 @@ describe(`Module: Proposals`, () => {
     let { actions } = module
     jest.useFakeTimers()
 
-    const rootState = {
-      wallet: {
-        address: addresses[0]
-      }
-    }
     let dispatch = jest.fn()
-    proposals.forEach(async (proposal, i) => {
+    Object.values(proposals).forEach(async (proposal, i) => {
```

Comment:
same here

Reply:
here I would like to keep the values array to have the index, we are using it on line 189